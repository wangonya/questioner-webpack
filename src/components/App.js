import React from "react";
import TopNav from "./Navbar";
function App() {
  return (
    <div>
      <TopNav />
    </div>
  );
}

export default App;
