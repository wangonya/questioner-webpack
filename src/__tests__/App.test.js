import React from "react";
import { shallow } from "enzyme";

import App from "./../components/App";

describe("<App /> root component", () => {
  test("renders without crashing", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });

  test("renders <Navbar /> component", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find("Connect(TopNav)")).toHaveLength(1);
  });
});
