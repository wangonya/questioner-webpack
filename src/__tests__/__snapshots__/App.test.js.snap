// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> root component renders without crashing 1`] = `ShallowWrapper {}`;
